apps/src/templates/studioHomepages/JoinSectionNotifications.jsx

```diff
@@ -86,6 +88,17 @@ const JoinSectionOwnedNotification = ({sectionId}) => (
 JoinSectionOwnedNotification.propTypes =
   JoinSectionNotFoundNotification.propTypes;
 
+const JoinSectionCaptchaNotification = ({sectionId}) => (
```

Comment:
This makes me a little nervous. Seeing this notification would be a big disruption for a classroom if it means the student can't join the section. I'm not sure if there is a better way to handle this however. 

One possibility - give the user a way to self-correct. i.e. they see this if they a) try incorrectly 3 times _and_ b) google recaptcha is inaccessible for some reason. Perhaps in this case we just let them keep trying. If that's not a possible option, we could tell the how what they can do to fix it i.e. "The reCAPTCHA service is down right now and you've tried joining the section too many times. Wait a few minutes to try again and complete the reCAPTCHA when it is presented. If you keep seeing this message, contact support@code.org" - not quite like that, but hopefully that gets the idea across?